### ISSUE TITLE
COCOSSD: WEBGL_PACK=true leads to different results vs Python

### ISSUE THREAD

**Issue by msektrier** [tensorflow/tfjs#1652]
#### TensorFlow.js version

Model was converted to Model.JSON via 

> tb-nightly (1.14.0a20190603)
> tensorflow-estimator-2.0-preview (1.14.0.dev2019060300)
> tensorflow-hub (0.3.0)
> tensorflowjs (1.1.2)
> termcolor (1.1.0)
> tf-nightly-2.0-preview (2.0.0.dev20190603)

#### Browser version
Chrome Version 74.0.3729.169 (Offizieller Build) (64-Bit)

#### Describe the problem or feature request
When we execute the model in Python with TF 2.0 we get different results than in TFJS. We use the latest version from [TFJS ](https://cdn.jsdelivr.net/npm/@tensorflow/tfjs/dist/tf.min.js).
In the output tensor "**detection_boxes**" it seems everything is screwed. 1st and second row are fine and in between I can find other equal rows. 

![image](https://user-images.githubusercontent.com/49950048/58872101-9bac0f00-86c3-11e9-9766-2c30f4df0da7.png)

and in Python with TF 2.0 we get 

![image](https://user-images.githubusercontent.com/49950048/58872238-d9109c80-86c3-11e9-8917-5c129066b57b.png)

#### Code to reproduce the bug / link to feature request

I prepared a complete HTML skeleton. By calling the index.html it is loading the model.json and predicts. 

https://share.infinkon.de/index.php/s/tD9QJ98NwJp2jd6

In the folder **"tf/ssdlite_saved_model_u2/Saved_Model_ssdlite_v2_1class"** you will also find the original saved model we used as a source for the **tensorflowjs_converter**.

**Please note**: only detection_boxes is available. 

many thanks
Sebastian

**Comment by joao-carvalheira** [comment]
I think I'm having the same problem hmm... I made a landmark detection model and it works fine in tensorflow but when I use it in the browser (tfjs) it gives me completely wrong results. Like instead of receiving normal coordinate values like [123.3..., 94.2...], I get values around these [-97.64..., 220.22...] like if the video was almost frozen and also giving negative coordinates.
If I only change the string that points to the previous version of the same model it just works. I think I didn't change much of the structure except for BatchNormalization...

**Comment by dsmilkov** [comment]
Hi @msektrier, I noticed that the image you used in the app is high resolution (4032x3024). Do the results in Python match if you feed a smaller res image (400x300 e.g.)? This will help us narrow down the problem

**Comment by dsmilkov** [comment]
Update: The problem is due to internals of TF.js. In the meantime, the workaround is to add `tf.ENV.set('WEBGL_PACK', false);` in the `run()` method before you load the graph and the results should match Python.

@annxingyuan Can you take a look at this problem? Turns out when `WEBGL_PACK=false`, the results in the browser match Python, but `WEBGL_PACK=true` leads to different results.

**Comment by annxingyuan** [comment]
I've tested the repo and it looks like this is solved by https://github.com/tensorflow/tfjs-core/pull/1796

**Comment by dsmilkov** [comment]
Great! For our users, once we release a new version of TF.js you should remove the temporary workaround and use the latest
